code/triangle/View/src/main/resources/ru/unn/agile/triangle/view/Calculator.fxml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import javafx.geometry.*?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import ru.unn.agile.triangle.viewmodel.*?>
+
+<AnchorPane xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="ru.unn.agile.triangle.view.Calculator">
```

Comment:
Try to fix displacement of Bx and By fields (one pixel, but its very painful for perfectionists)